lib/Checks.groovy

```diff
@@ -1,9 +1,29 @@
+import org.yaml.snakeyaml.Yaml
+
 /*
  * This file holds several functions used to perform standard checks for the nf-core pipeline template.
  */
 
 class Checks {
 
+    static void check_conda_channels() {
+        def message = "There's a problem with your conda configuration. " +
+                      "Please refer to  https://bioconda.github.io/user/install.html#set-up-channels " + 
+                      "to learn how to set-up the conda-forge and bioconda channels correctly. "
+                      "NB: the order of the channels matters!"
+
+        Yaml parser = new Yaml()
+        def channels = parser.load("conda config --show channels".execute().text).channels
```

Comment:
Nice! Can the output of this command be empty? If so, will this still work?

Reply:
I added try/catch to handle edge cases (e.g. conda not installed)

Reply:
Great!